lib/manager/nuget/index.ts

```diff
@@ -10,6 +10,7 @@ export const defaultConfig = {
     '\\.(?:cs|fs|vb)proj$',
     '\\.(?:props|targets)$',
     '\\.config\\/dotnet-tools\\.json$',
+    '^dotnet-tools\\.json$',
```

Comment:
```suggestion
    '(?:\\.config\\/)?dotnet-tools\\.json$',
```

Simplify 🙃

Reply:
That would find a dotnet-tools.json-file anywhere right? Would be fine with me, then we could skip the .config-part of the regex

Reply:
Is it defined to only be in two locations, or could it be anywhere?

Reply:
As I'm writing this I realize that dotnet-tools.json can be in any location, since the `dotnet tool` command doesn't have to be run with current directory as the repository root. I'm not sure if renovate supports that though, as I haven't tested.

👇 With the (faulty?) assumption that `dotnet tool` is only run in repository root

dotnet reads the following locations:

- `.config/dotnet-tools.json` (in repo, default location, supported by renovate)
- `dotnet-tools.json` (in repo)
- `../dotnet-tools.json` (outside repo)
- `../../dotnet-tools.json` (outside repo)
- etc

So the two valid locations in the repository are  `.config/dotnet-tools.json` and  `dotnet-tools.json`

Reply:
Let's look in any location though. There's very little harm, considering how unique the filename is

Reply:
Maybe the expression can be simplified further

```suggestion
    '(^|//)dotnet-tools\\.json$',
```

dotnet-tools.json anywhere or i repo root.

What do you think @rarkins ?

Reply:
`//` should be `\/` 😉

Reply:
> `//` should be `\/` 😉

I copied that from the expression below 🤷 I can change it

Reply:
No need to escape `/ ` it seems, I just need to remove the duplicate?

```suggestion
    '(^|/)dotnet-tools\\.json$',
```

Reply:
```suggestion
    ''(^|\/)dotnet-tools\\.json$',
```

Please escape for safty:
![image](https://user-images.githubusercontent.com/1798109/140479095-8c5f8305-b0a0-46b0-8188-adf4bf6513bc.png)
![image](https://user-images.githubusercontent.com/1798109/140479160-d6c3709e-d7f5-48dd-8a09-11267baaa5e2.png)